content/messaging/customer-chat-api/changelog/index.mdx

```diff
@@ -9,6 +9,16 @@ desc: "Changelog of the Customer Chat API"
 
 This document is the record of all the changes in the **Customer Chat API**, Web and RTM, starting from **version 3.0**.
 
+## [v3.3] - Developer Preview
+
+### Changed
+- Parameters have been either ranamed or removed in following methods:
```

Comment:
I think we should be more specific, for example:

- The `license_id` and `group_id` parameters were removed in the [**List Group Properties**](/messaging/customer-chat-api/v3.3/#list-group-properties) and [**List License Properties**](/messaging/customer-chat-api/v3.3/#list-license-properties) methods.
- The `chat_id` parameter was renamed to `id` in the following methods:
    - Deactivate Chat
    - List Group Properties
    - etc.